test-local/cli-doctor-collect-only.test.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+const fs = require('fs')
+const path = require('path')
+const test = require('tap').test
+const cli = require('../test/cli.js')
+
+test('clinic doctor --collect-only --dest \'./foo\' - no issues', function (t) {
+  cli({ relayStderr: false }, [
+    'clinic', 'doctor', '--collect-only', '--dest', './foo',
+    '--', 'node', '-e', 'setTimeout(() => {}, 300)'
```

Comment:
Suggestion: we do not have to run the application for at least 300 ms and could use a significantly lower value.

Reply:
I agree but I took it for consistency from https://github.com/nearform/node-clinic/blob/master/test-local/cli-flame-collect-only.test.js#L11. Shall we keep them aligned around `100`?

Reply:
I would likely just use a value as low as 25 or so. It does not matter how much we actually profile as long as it's enough for the test to pass.

And yes, the other value should likely also be reduced.